src/components/dashboard.js

```diff
@@ -1,155 +1,206 @@
 import React from "react";
 import {
-  Container,
   Segment,
   SegmentGroup,
   Header,
-  Checkbox,
-  Label,
-  Table,
   Dimmer,
   Loader,
   Message
 } from "semantic-ui-react";
 import DatePicker from "react-datepicker";
-// import JSONPretty from "react-json-pretty";
+import moment from 'moment'
+// import { render } from 'react-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+//import $ from 'jquery';
+// import Popper from 'popper.js';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
```

Comment:
Move this import also to `App.js` like that you'll have access to it everywhere in the child components.